src/SVGSkin.js

```diff
@@ -58,6 +59,35 @@ class SVGSkin extends Skin {
         return this._svgRenderer.size;
     }
 
+    useNearest (scale, drawable) {
+        // If the effect bits for mosaic, pixelate, whirl, or fisheye are set, use linear
+        if ((drawable.enabledEffects & (
+            ShaderManager.EFFECT_INFO.fisheye.mask |
+            ShaderManager.EFFECT_INFO.whirl.mask |
+            ShaderManager.EFFECT_INFO.pixelate.mask |
+            ShaderManager.EFFECT_INFO.mosaic.mask
+        )) !== 0) {
+            return false;
+        }
+
+        // We can't use nearest neighbor unless we are a multiple of 90 rotation
+        if (drawable._direction % 90 !== 0) {
+            return false;
+        }
```

Comment:
What do you think of keeping the drawable.useNearest function, having these lines stay there, and then having it return skin.useNearest? Then both drawable and skin get to have a say in whether we useNearest

Reply:
These lines are very `SVGSkin`-specific--it kinda sucks that `SVGSkin` depends on its parent drawable to know whether it should use nearest-neighbor interpolation but I'm not sure what else can be done.